### ISSUE TITLE
Terminal not filling whole height after keypress

### ISSUE THREAD

**Issue by frevds** [microsoft/vscode#108958]
Issue Type: <b>Bug</b>

Terminal window somehow does not fill the whole line but seems to have some invisible status bar area. 
Clicking on the bottom part of the console does not activate the console then. 
This is not always the case though. Switching to another pane such as the Debug Console and going back, the Terminal fills the whole height again, but only until I press the Spacebar key (any other key or Enter does not cause the issue). 

![image](https://user-images.githubusercontent.com/5341447/96514817-83ab4000-1264-11eb-97ce-c2a404031506.png)

![ezgif-7-371ff6f34183](https://user-images.githubusercontent.com/5341447/96515982-90c92e80-1266-11eb-9be4-e8da65471ab0.gif)

VS Code version: Code 1.50.1 (d2e414d9e4239a252d1ab117bd7067f125afd80a, 2020-10-13T15:06:15.712Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E3-1505M v6 @ 3.00GHz (8 x 3000)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.86GB (20.30GB free)|
|Process Argv|-n --crash-reporter-id 9d649195-38d9-40dc-ade7-595f806409f4|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
xml|Dot|2.5.1
vscode-azureappservice|ms-|0.19.0
vscode-azurefunctions|ms-|0.24.1
azure-account|ms-|0.9.3
references-view|ms-|0.0.70
material-icon-theme|PKi|4.3.0
@shareflex/shareflex|Por|1.1.0
gitflow|vec|1.2.1
vscodeintellicode|Vis|1.2.10
markdown-pdf|yza|1.4.4

</details>

**Comment by frevds** [comment]
PS: This seems to have something to do with word-wrap of the lines above (see video). If the window is maximized this issue disappears, although not completely. Apparently the height of the console is calculated explicitly rather than automatically by css.

**Comment by Tyriar** [comment]
@frevds are you using the dom renderer? Check for the `terminal.integrated.rendererType` setting.

Could you see if setting `"terminal.integrated.rendererType": "experimentalWebgl"` fixes the issue?

**Comment by frevds** [comment]
@Tyriar Yes, I was using "dom". With experimentalWebgl I get the same issue when a scrollbar exists and I press space after a word, and if I press a lot of spaces I can even make the scrollbar move inwards once it exceeds the line (can't reproduce that with the dom-renderer though): 
![image](https://user-images.githubusercontent.com/5341447/97753874-cd164d80-1af6-11eb-8a3b-f59c05478534.png)

The issue seems to be with the container, whose size is somehow calculated explicitly rather than by the browser. That might be done at the wrong time or using the wrong input.
